lib/bolt/cli.rb

```diff
@@ -45,6 +46,7 @@ class CLI
 
 Available actions are:
     show                             Show list of available tasks
+    show <task>                      Show documentation for task
```

Comment:
I'm not sure if this should be here or not.

Reply:
I think this is better than a single line and definitely better than not mentioning it.